ledger/tracker.go

```diff
@@ -562,6 +593,11 @@ func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) error {
 	start := time.Now()
 	ledgerCommitroundCount.Inc(nil)
 	err = tr.dbs.Transaction(func(ctx context.Context, tx trackerdb.TransactionScope) (err error) {
```

Comment:
Non-blocking question - is there any type of configured/default timeout that results in an error? I assume the answer is panic, but sanity checking.